Xamarin.Forms.Platform.Android/Extensions/TextViewExtensions.cs

```diff
@@ -7,41 +8,65 @@ namespace Xamarin.Forms.Platform.Android
 {
 	internal static class TextViewExtensions
 	{
-		public static void SetLineBreakMode(this TextView textView, LineBreakMode lineBreakMode)
+		public static void SetMaxLines(this TextView textView, Label label)
 		{
+			var maxLines = label.MaxLines;
+
+			if (maxLines == (int)Label.MaxLinesProperty.DefaultValue)
+			{
+				// MaxLines has not been explicitly set, so just let it be whatever was set by LineBreakMode
+				return;
+			}
+
+			textView.SetMaxLines(maxLines);	
+		}
+
+		static void SetMaxLines(this TextView textView, Label label, int lines)
+		{
+			// If the Label's MaxLines has been explicitly set, we should not set it here
+			if (label.MaxLines != (int)Label.MaxLinesProperty.DefaultValue)
+			{
+				return;
+			}
+
+			textView.SetMaxLines(lines);
```

Comment:
Doesn't this create an issue where Core and native properties are no longer in sync? LineBreakMode controls native maxLines, but MaxLines in Core is not aware of the change.

Reply:
It does, but I cannot immediately come up with any negative consequences from it.